### PR TITLE
BH-407: consuming queue is now non blocking

### PR DESCRIPTION
**Pull request by greg-akeneo**
Make job consumer non blocking when reading the job queue.
This is important when processes have to terminate gently to be able to poll the queue and check if a signal has been emitted.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Tool/Bundle/BatchQueueBundle/Command/JobQueueConsumerCommand.php [hunk 1]
```diff
@@ -87,61 +88,71 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $whitelistedJobInstanceCodes = $input->getOption('job');
-        $blacklistedJobInstanceCodes = $input->getOption('blacklisted-job');
+        /** @var JobQueueConsumerConfiguration */
+        $configuration = (new JobQueueConsumerConfiguration())
+            ->setTimeToLive(5)
```
Comment (BitOne) [inline src/Akeneo/Tool/Bundle/BatchQueueBundle/Command/JobQueueConsumerCommand.php:93]: Magic numbers. Can you use constants to identify what they are and their unit?

**Review by BitOne — APPROVED** [review]
GTM after fix

**Comment by greg-akeneo** [comment]
make the job ID to appear with the `-v` option

File: src/Akeneo/Tool/Bundle/BatchQueueBundle/Command/JobQueueConsumerCommand.php (hunk 1 above)
Comment (greg-akeneo) [reply src/Akeneo/Tool/Bundle/BatchQueueBundle/Command/JobQueueConsumerCommand.php]: done in a32f8beaec

**Review by mr-prud — APPROVED** [review]

**Pull request merged**